docs/resources/metal_device.md

```diff
@@ -154,7 +154,7 @@ resource "equinix_metal_device" "pxe1" {
 
 ### Required
 
-- `operating_system` (String) The operating system slug. To find the slug, or visit [Operating Systems API docs](https://metal.equinix.com/developers/api/operatingsystems), set your API auth token in the top of the page and see JSON from the API response.  By default, changing this attribute will cause your device to be deleted and recreated.  If `reinstall` is enabled, the device will be updated in-place instead of recreated.
+- `operating_system` (String) The operating system slug. To find the slug, or visit [Operating Systems API docs](https://deploy.equinix.com/developers/docs/metal/operating-systems/supported/s), set your API auth token in the top of the page and see JSON from the API response.  By default, changing this attribute will cause your device to be deleted and recreated.  If `reinstall` is enabled, the device will be updated in-place instead of recreated.
```

Comment:
```suggestion
- `operating_system` (String) The operating system slug. To find the slug, or visit [Operating Systems API docs](https://deploy.equinix.com/developers/docs/metal/operating-systems/supported/), set your API auth token in the top of the page and see JSON from the API response.  By default, changing this attribute will cause your device to be deleted and recreated.  If `reinstall` is enabled, the device will be updated in-place instead of recreated.
```